css/amp.css

```diff
@@ -148,6 +148,8 @@ i-amp-sizer {
 
 .-amp-element > [placeholder] {
   display: block;
+  /** Background is important to hide all other placeholders below it. */
+  background-color: #efefef;
```

Comment:
We can stick with a white background and maybe make this somewhat less specific so it would allow other selectors to override this correctly.